frontend/package.json

```diff
@@ -56,6 +56,7 @@
 		"eslint-plugin-svelte": "^2.39.0",
 		"globals": "^13.24.0",
 		"graphql": "^16.8.1",
+    "json5": "^2.2.3",
```

Comment:
This ended up using spaces, but every other line uses tabs. Either your editor doesn't have the EditorConfig plugin installed, or else (more likely) we don't have a definition for package.json in our `.editorconfig` file to ensure that the editor is set to use tabs for indentation in this file. Fixing the .editorconfig is out of scope for this PR, but please do make sure this line is indented with a tab character to match the rest of the file. Otherwise, as you may see here, the diff starts to look ugly.